recipe/upload_or_check_non_existence.py

```diff
@@ -22,7 +22,14 @@ def built_distribution_already_exists(cli, meta, owner):
 
     """
     distro_name = '{}/{}.tar.bz2'.format(conda.config.subdir, meta.dist())
-    fname = bldpkg_path(meta)
+    try:
+        # conda-build >= 2.0.0 takes a config argument
+        from conda_build.config import Config
+        config = Config()
```

Comment:
This line could be moved outside the try block, although I can understand why it would be included here.